lib/pages/home_page.dart

```diff
@@ -10,39 +15,65 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
-  int _counter = 0;
+  final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
+
+  List<AppBarAndDrawerItemModel> pageSectionsList = [];
```

Comment:
Try to make it final. As far as I know, it is not a problem to add an item to a final list.